documentation/publish/self-distribution.md

```diff
@@ -52,10 +52,10 @@ For more information on how to submit an add-on for distribution on AMO or self-
 
 When you choose to distribute an add-on yourself, they can be installed using the following methods:
 
-- Web download—make your extension available on a suitable web accessible server and when the user downloads the signed add-on file Firefox will install it.
-- [Sideloading add-ons](/documentation/publish/distribute-sideloading/)—enables a user to install an add-on using an XPI file saved on their computer.
-- [Add-ons for use with a desktop app](/documentation/publish/distribute-for-desktop-apps/)—this section describes the best practices for delivering an extension for use with a desktop application.
-- [Add-ons in an enterprise environment](/documentation/enterprise/enterprise-distribution/)—this page discusses the use of signed compared to unsigned extensions, installation options, the Firefox settings affecting installation, and including add-ons with a custom Firefox install package.
+- [Web download](https://github.com/piculet/extension-workshop/blob/master/documentation/publish/submitting-an-add-on.md#self-distribution)—make your extension available on a suitable web accessible server and when the user downloads the signed add-on file Firefox will install it.
```

Comment:
What is this? Can you please copy the accurate link mentioned in issue by @caitmeunster

Reply:
Really sorry about that, it was a careless mistake